packages/patternfly-4/react-core/src/components/LoginPage/LoginForm.tsx

```diff
@@ -71,7 +71,7 @@ export const LoginForm: React.FunctionComponent<LoginFormProps> = ({
       {helperText}
     </FormHelperText>
     <FormGroup label={usernameLabel} isRequired isValid={isValidUsername} fieldId="pf-login-username-id">
-      <TextInput
+      <TextInput autoFocus
```

Comment:
Can we add a `noFocus` prop in case a consumer doesn't want this new behaviour? Otherwise, nice find!!!